SConstruct

```diff
@@ -607,6 +608,10 @@ env['ROC_MODULES'] = [
 
 env['ROC_TARGETS'] = []
 
+env['PKG_CONFIG_DEPS'] = []
+
+env['PKG_CONFIG_LIBS'] = []
```

Comment:
Can we hide initialization of these variables inside scons plugin? E.g. automatically init them in AddPkgConfigDependency and AddPkgConfigLibs.